src/shared/transport.ts

```diff
@@ -41,4 +41,6 @@ export interface Transport {
    * Callback for when a message (request or response) is received over the connection.
    */
   onmessage?: (message: JSONRPCMessage) => void;
+
+  sessionId?: string;
```

Comment:
Can you please add a docstring here?

Reply:
You bet, I'll take a look this afternoon

Reply:
Hey sir - added a test and a doc string, let me know if you need anything else!